pkg/app/piped/planpreview/handler.go

```diff
@@ -262,6 +261,7 @@ func (h *Handler) handleCommand(ctx context.Context, cmd model.ReportableCommand
 		if err != nil {
 			logger.Error("failed to marshal command result", zap.Error(err))
 		}
+
 		if err := cmd.Report(ctx, model.CommandStatus_COMMAND_FAILED, nil, output); err != nil {
```

Comment:
With the current code, this `Report` function will not be executed when the command context's deadline passes.

Reply:
Ah, got it. Nice catch 👍